app/controllers/agents/dry_runs_controller.rb

```diff
@@ -8,6 +8,8 @@ def index
                 elsif params[:source_ids]
                   Event.where(agent_id: current_user.agents.where(id: params[:source_ids]).pluck(:id))
                        .order("id DESC").limit(5)
+                else
```

Comment:
Because of [deep_munge](https://github.com/rails/rails/issues/13420) an empty array in the params will be converted to `nil` and that results in `@events` being undefined.